### PR TITLE
Updates to documentation. 

### PR DESCRIPTION
**Pull request by straithe**
Updated documentation to replace incorrect link.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
 1) Firmware for a compatible CAN-transceiver
 2) Python client to talk to the tool
 
-it is a lot like the architecture for RfCat, and we may make it moreso.  currently we're sticking with the original design by Matt and team.
+CanCat is a lot like the architecture for RfCat, and we may make it moreso. Currently we're sticking with the original design by Matt and team.
 
-currently supported CAN-transceiver combinations:
+The currently supported CAN-transceiver combinations:
 * Arduino with SeeedStudio's CANBUS Shield
 * Arduino DUE with Togglebit.net's CAN shield
```
Comment (atlas0fd00m) [inline README.md]: sorry to glom onto your PR, but would you mind adding in the Macchina M2 (with a link to the macchina.cc page for both under-hood and under-dash versions)?

Comment (straithe) [reply README.md]: Ok.

**Comment by straithe** [comment]
Please let me know if this update is satisfactory.

File: README.md [hunk 2]
```diff
@@ -288,7 +288,6 @@ $ citm.printCanMsgs() # Prints all CAN messages
 $ citm.printCanMsgsIso() # prints all CAN messages received on the Isolation network
 ```
 
-Placing a bookmark places a bookmark simultaneously on both the Isolation information and the aggregate information.
+Placing a bookmark places a bookmark simultaneously on both the isolation information and the aggregate information.
```
Comment (atlas0fd00m) [inline README.md]: actually, this capital has a purpose for the moment.  let's leave it alone.  we'll talk more about the meanings later, and probably change out the terminology a bit in the future.

Comment (straithe) [reply README.md]: Ok. Updated.

**Comment by straithe** [comment]
Thank you!

**Pull request merged**